src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/PrisonServiceClient.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.personrecord.client
+
+import org.springframework.cloud.openfeign.FeignClient
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner.PrisonerDetails
+import uk.gov.justice.digital.hmpps.personrecord.config.PrisonServiceOAuth2Config
+
+@FeignClient(
+  name = "prison-service",
+  url = "\${prison-service.base-url}",
+  configuration = [PrisonServiceOAuth2Config::class],
+)
+interface PrisonServiceClient {
+
+  @GetMapping(value = ["\${prison-service.prisoner-details}/{prisonerNumber}"])
```

Comment:
I know it's a pattern we have elsewhere but I am not sure why we get the path from a property. It does not change per environment. `prison-service.prisoner-details` is always `/offender`

Reply:
It's a recommended practice. For easy maintenance, future-proofing, transparency and consistency.

Reply:
It is certainly consistent :-)!